x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/create_signals.route.test.ts

```diff
@@ -157,7 +161,8 @@ describe('create_signals', () => {
           type: 'something-made-up', // This is a made up type that causes the 400
           from: 'now-6m',
           to: 'now',
-          kql: 'user.name: root or user.name: admin',
+          query: 'user.name: root or user.name: admin',
+          language: 'kuery',
```

Comment:
For example, I think we could replace https://github.com/elastic/kibana/blob/17ad88924aa1ee28172012c3acbc5f6e96543050/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/create_signals.route.test.ts#L72-L88

with 

```
const payload = typicalPayload();
delete payload.id;
const request: ServerInjectOptions = {
   method: 'POST',
   url: '/api/siem/signals',
   payload,
 };```

Reply:
Ha! Look at that file name. I accidentally added an extra dot and called it `create_signals.route.test.ts` rather than `create_signals_route.test.ts`

That test won't run until I change the name to not have two dots in it.

Reply:
Yeah I changed it to use the payload with an ES2015+/ES6+ style omit and that seems to work out as well as fixing my file name.